prisma/dev-seed-utils/permits.ts

```diff
@@ -3,19 +3,40 @@
 import prisma from '../index'; // Prisma client
 
 // Seed data
-const permits = [{ rcdPermitId: 1, applicantId: 1, applicationId: 1 }];
+const permits = [
+  {
+    rcdPermitId: 1,
+    applicantId: 1,
+    applicationId: 1,
+    expiryDate: new Date(2025, 1, 1).toISOString(),
+    active: true,
+  },
```

Comment:
Thanks @EmilioMena for taking the time to add extra seeding data! Will help us a lot for testing